app/views/admin/certifications/_table_demotions.html.erb

```diff
@@ -15,15 +15,25 @@
 
     <tbody>
     <% @demotions.each do |demotion| %>
+      <% has_training_session = demotion.training_session.present? %>
```

Comment:
I would remove this and have the <% if demotion.training_session.present? %> go one level higher so we can put the <td> inside it and the else would just be 3 empty <td>